corehq/apps/commtrack/tests/test_rebuild.py

```diff
@@ -90,16 +90,14 @@ def test_inferred(self):
 
     @run_with_all_backends
     def test_case_actions(self):
-        """
-        make sure that when a case is rebuilt (using rebuild_case)
-        stock transactions show up as well
-        """
+        # make sure that when a case is rebuilt (using rebuild_case)
+        # stock transactions show up as well
         form_id = self._submit_ledgers(LEDGER_BLOCKS_SIMPLE)
         case_id = self.case.case_id
         rebuild_case_from_forms(self.domain, case_id, RebuildWithReason(reason='test'))
         case = CaseAccessors(self.domain).get_case(self.case.case_id)
         self.assertEqual(case.xform_ids[1:], [form_id])
-        self.assertEqual(case.actions[1].form_id, form_id)
+        self.assertTrue(form_id in [action.form_id for action in case.actions])
```

Comment:
no need to change, but could use `self.assertIn(form_id, [action.form_id for action in case.actions])`